archunit/src/main/java/com/tngtech/archunit/core/importer/ImportOption.java

```diff
@@ -74,21 +74,23 @@ public boolean includes(Location location) {
      * Thus it is just a best guess, how tests can be identified,
      * in other environments, it might be necessary, to implement the correct {@link ImportOption} yourself.
      */
-    final class DoNotIncludeTests implements ImportOption {
-        private static final Pattern MAVEN_PATTERN = Pattern.compile(".*/target/test-classes/.*");
-        private static final Pattern GRADLE_PATTERN = Pattern.compile(".*/build/classes/([^/]+/)?test/.*");
-        private static final Pattern INTELLIJ_PATTERN = Pattern.compile(".*/out/test/classes/.*");
+    final class DoNotIncludeTests extends LocationPatternImportOption {
 
-        private static final Set<Pattern> EXCLUDED_PATTERN = ImmutableSet.of(MAVEN_PATTERN, GRADLE_PATTERN, INTELLIJ_PATTERN);
-
-        @Override
-        public boolean includes(Location location) {
-            for (Pattern pattern : EXCLUDED_PATTERN) {
-                if (location.matches(pattern)) {
-                    return false;
-                }
-            }
-            return true;
+        public DoNotIncludeTests() {
+          super(TESTS_LOCATIONS, true);
+        }
+    }
+    
+    /**
+     * Import option to define rules only for Tests.
+     * See {@link DoNotIncludeTests} for limitations if tests identification.
+     */
+    final class OnlyTests extends LocationPatternImportOption {
+       
+        private static final Set<Pattern> INCLUDED_PATTERN = ImmutableSet.of(MAVEN_PATTERN, GRADLE_PATTERN, INTELLIJ_PATTERN);
```

Comment:
This constant is unused, isn't it?

Reply:
8b715662339d41d8964862bf82f759ca927771bb